ui/src/components/Organization/List.tsx

```diff
@@ -7,23 +7,35 @@ import { Organization } from 'src/entities/Organizations';
 const c = classNames.bind(style);
 
 interface Props {
+  isFetching?: boolean;
   organizations: Organization[];
 }
 
 const OrgItem: React.FC<Organization> = (o) => (
-  <div className={c('container')} key={o.id}>
+  <div className={c('organization')} key={o.id}>
     <div className={c('fallback-icon')}>
       <FallbackIcon logo={o.square_logo} name={o.name} />
     </div>
-    <div className={c('organization-container')}>
+    <div className={c('organization-details')}>
       <span className={c('name')}>{o.name}</span>
       <span className={c('count')}>{o.employees_count} Employees</span>
     </div>
   </div>
 );
 
+const Skeleton = () => (
+  <div className={c('skeleton')}>
+    {OrgItem({ banner: '', id: '', square_logo: '', name: 'ABC', employees_count: 0 })}
+  </div>
+);
+
 const List: React.FC<Props> = (p) => {
-  return <div className={c('organization')}>{p.organizations.map(OrgItem)}</div>;
+  return (
+    <div className={c('container')}>
+      {p.isFetching && <Skeleton />}
+      {p.organizations.map(OrgItem)}
```

Comment:
In case there is an error while fetching organizations what is happening here. Is it notified somewhere?

Reply:
This component does not hold the responsibility of fetching organizations, hence don't need to care about errors that happen during fetching organizations. This is a pure component in regard of network communication. It's parent fetches the organizations, takes care of the errors, and pass them to this component to render the UI.